torch/onnx/_internal/fx/function_dispatcher.py

```diff
@@ -159,8 +160,19 @@ def _create_op_overload_to_exporter_key_table() -> (
     # TODO(justinchuby): Improve how the table is constructed.
     table: Dict[Union[torch._ops.OpOverload, Callable], str] = {}
 
+    # Some ops in `torch.ops.aten` are not discoverable through `dir(torch.ops.aten)`,
+    # but retrievable via explicit lookup.
+    # https://github.com/pytorch/pytorch/issues/99681
+    # This is a workaround to make sure we register ONNX symbolic functions for these.
```

Comment:
Does this way help us find aten_sym_size?

Reply:
Great question, it does :)

Reply:
```
>>> import torch
>>> "sym_size" in torch.ops.aten
False
>>> torch.ops.aten.sym_size
<OpOverloadPacket(op='aten.sym_size')>
```

Reply:
Should we move sym_size to torchlib

Reply:
I am happy to include as many as we see

Reply:
Curious if it has opinfo. I wonder if it means higher maintenance cost than rest of 'normal' ops. I don't have strong opinion either way.

Reply:
What does it mean "moving to torchlib"? Isn't it in onnx-script core.py?

Reply:
thanks @titaiwangms, case closed!

Reply:
😨

Reply:
Ahh sorry I mistook it for getitem